Tue Mar 20 03:49:24 FET 2012  Adam Vogt <vogt.adam@gmail.com>
  * Address versioning problems related to X11 1.6 release.
  
  Bump version to 0.10.1 since cabal uses hackage dependencies even when the
  locally installed package differs.
  
  Allow X11-1.6 dependency.
diff -rN -u old-xmonad/xmonad.cabal new-xmonad/xmonad.cabal
--- old-xmonad/xmonad.cabal	2012-05-06 12:20:26.788767220 +0300
+++ new-xmonad/xmonad.cabal	2012-05-06 12:20:26.818766844 +0300
@@ -46,7 +46,7 @@
         build-depends: base < 5 && >=3, containers, directory, process, filepath, extensible-exceptions
     else
         build-depends: base < 3
-    build-depends: X11>=1.5.0.0 && < 1.6, mtl, unix,
+    build-depends: X11>=1.5 && < 1.7, mtl, unix,
                    utf8-string >= 0.3 && < 0.4
 
     if true
